### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by RafaelLinux** [mediadrop/mediadrop#251]
Hello

I found this software very interesting, however, is barely updated .... is this project dead??

Thank you

**Comment by gregfr** [comment]
One year later, no answer... I guess the project is dead :(

**Comment by alexander01202** [comment]
> One year later, no answer... I guess the project is dead :(

Almost two year later, no answer.... Surely the project is dead :(

**Comment by toxinliquid** [comment]
another 2 years , it is surely dead.

**Comment by dewhisna** [comment]
There's still a few people, such as myself, using it, but I haven't seen any development here in a long time.  It still [barely] works, but there's a lot of caveats...

It still uses Python 2.7 and would take a bit of work to move to the 3.x world.  It's also stuck on SQLAlchemy 0.7.10 or older, as the API changes in SQLAlchemy 0.8.x+ would require class_mapper code changes.  And it's also stuck on older MySQL versions and doesn't work on MySQL 8.0, for example, as that would require an even newer SQLAlchemy which would also need the event callbacks into MediaDrop reworked, which isn't totally trivial.

I recently updated the server where I've been running it from Bionic to Focal and it took quite a bit of headbanging to keep it running without extensive code work.  It included having to pin MySQL 5.7 and manually dumping and reloading all of the MySQL data using `mysqldump` because the `dist-upgrade` mangled up the database in the move from 5.7 to 8.0 during the upgrade and it was no longer possible to just downgrade without blowing away all the data and reloading it.

I'm not sure it would survive an upgrade to Jammy.  I've started searching around for an alternative.  Though I suppose that when Focal EOL is here, I could run a Focal Docker container with MediaDrop hosted out of it via a reverse-proxy on the backend, so that it can stay isolated.

**Comment by RafaelLinux** [comment]
I think GITHUB should tag this as obsolete yet.

**Comment by Sajiyah-Salat** [comment]
Hello @RafaelLinux are you using this project. I wanted to use it an ngo project. for storing recording of classes and all. Shoudl I use it.

**Comment by RafaelLinux** [comment]
No. I was interested in finding software with features like the ones they publish on their website, but as the project seemed abandoned, I haven't even tried it.

**Comment by odkr** [comment]
One alternative is <https://github.com/mediacms-io/mediacms>. But it's also maintained by a single developer. We have migrated to WordPress and the "Seriously Simple Podcasting" plugin, which offer somewhat equivalent features and are backed by a larger team.
